### PR TITLE
Add options to hide different parts of embed UI (parameters, title, link to query)

### PR DESCRIPTION
**Pull request by arikfr**
## What type of PR is this? (check all applicable)

- [x] Feature

## Description

Add options to hide different parts of embed UI (parameters, title, link to query).

### CODE REVIEW

**Comment by ranbena** [comment]
@arikfr This is for snaps?

**Comment by arikfr** [comment]
@ranbena we might use it for snaps in the future, but this is actually for a customer who wanted cleaner embeds.

I think that later on we will add this as switches to the embed dialog to make it discoverable. But this was an easy way to deliver this.

File: client/app/components/queries/visualization-embed.html [hunk 1]
```diff
@@ -32,9 +32,9 @@ <h3>
           <span am-time-ago="$ctrl.queryResult.getUpdatedAt()" ng-if="!$ctrl.loading"></span>
           <rd-timer from="$ctrl.refreshStartedAt" ng-if="$ctrl.loading"></rd-timer>
         </a>
+        <span class="small visible-print"><i class="zmdi zmdi-time-restore"></i> {{$ctrl.queryResult.getUpdatedAt() | dateTime}} UTC</span>
```
Comment (ranbena) [inline client/app/components/queries/visualization-embed.html]: Please give this `margin-left: 5px` so it's not so close to the timeago text.

Comment (arikfr) [reply client/app/components/queries/visualization-embed.html]: What element should receive the extra margin?

Comment (ranbena) [reply client/app/components/queries/visualization-embed.html]:
```suggestion
        <span class="small visible-print m-l-5"><i class="zmdi zmdi-time-restore"></i> {{$ctrl.queryResult.getUpdatedAt() | dateTime}} UTC</span>
```

Comment (arikfr) [reply client/app/components/queries/visualization-embed.html]: But this shows up only when the other is hidden or I am misunderstanding something?

Comment (ranbena) [reply client/app/components/queries/visualization-embed.html]: Nope it's me who misunderstood how this works.

**Pull request merged**